### ISSUE TITLE
need help with this?

### ISSUE THREAD

**Issue by gyit918** [CerisWhite/CrAnberry#10]

**Comment by gyit918** [comment]
this happend

**Comment by gyit918** [comment]
![Screenshot 2021-10-01 7 36 18 PM](https://user-images.githubusercontent.com/85656283/135698178-2bb2777e-970f-4cff-b2cd-faf67f923411.png)

**Comment by vinaykharayat** [comment]
You are also pasting '$', please paste commands properly

**Comment by gyit918** [comment]
hu

**Comment by gyit918** [comment]
huh??

**Comment by gyit918** [comment]
wdym by that??

**Comment by gyit918** [comment]
wait how do i get rid of this dollar s

**Comment by vinaykharayat** [comment]
`setenforce Permissive`
This command should do the work.

**Comment by gyit918** [comment]
so what did that do??

**Comment by gyit918** [comment]
and when its permissive does that mean it has android root??

**Comment by gyit918** [comment]
its still enforcing

**Comment by gyit918** [comment]
![Screenshot 2021-10-01 9 29 13 PM](https://user-images.githubusercontent.com/85656283/135701025-d9571e4e-51ae-4f34-95da-19b122d392b9.png)

**Comment by gyit918** [comment]
its not doing anything

**Comment by gyit918** [comment]
![Screenshot 2021-10-01 9 33 31 PM](https://user-images.githubusercontent.com/85656283/135701119-0856e20b-b486-4a98-9aff-78a7cc3cf3b9.png)

**Comment by gyit918** [comment]
you know what ima just do this all over again

**Comment by gyit918** [comment]
how do i check for root

**Comment by gyit918** [comment]
dude you there

**Comment by gyit918** [comment]
this isnt working

**Comment by gyit918** [comment]
no it isnt i have a chromebook 360

**Comment by gyit918** [comment]
version 93

**Comment by gyit918** [comment]
well its saying its enforcing why??

**Comment by gyit918** [comment]
i did it all right

**Comment by gyit918** [comment]
did i need to remove roofts verifaction

**Comment by vinaykharayat** [comment]
Did you run the cranberry.sh and rooted successfully?

**Comment by gyit918** [comment]
> Did you run the cranberry.sh and rooted successfully?

idk

**Comment by gyit918** [comment]
it said for the last line reboot and enjoy

**Comment by gyit918** [comment]
but its still enforcing

**Comment by vinaykharayat** [comment]
Rooting does not have anything to do with SELinux enforcing or permissive. Although you will have restrictions on apps that modifies your system files like lucky patcher.
To make it simple for you please tell me for what reason you need to root your chromebook?

**Comment by gyit918** [comment]
> Rooting does not have anything to do with SELinux enforcing or permissive. Although you will have restrictions on apps that modifies your system files like lucky patcher. To make it simple for you please tell me for what reason you need to root your chromebook?

its something i wanted to do a long time thats all

**Comment by gyit918** [comment]
the reason i wanted to root is bc

**Comment by gyit918** [comment]
for something

**Comment by gyit918** [comment]
> Rooting does not have anything to do with SELinux enforcing or permissive. Although you will have restrictions on apps that modifies your system files like lucky patcher. To make it simple for you please tell me for what reason you need to root your chromebook?

then why does it matter for se linux

**Comment by gyit918** [comment]
> Rooting does not have anything to do with SELinux enforcing or permissive. Although you will have restrictions on apps that modifies your system files like lucky patcher. To make it simple for you please tell me for what reason you need to root your chromebook?

**Comment by gyit918** [comment]
so how do i give apps root permission then

**Comment by supechicken** [comment]
So can you puts all in a comment and don’t split them to separate comments

**Comment by vinaykharayat** [comment]
> > Rooting does not have anything to do with SELinux enforcing or permissive. Although you will have restrictions on apps that modifies your system files like lucky patcher. To make it simple for you please tell me for what reason you need to root your chromebook?
> 
> its something i wanted to do a long time thats all

If you don't know why you need to root then I would suggest not to root. You are trying to do something that you don't understand and you might end up messing up your system. Please don't root until you have a solid reason to root. 
Have a nice day!

**Comment by gyit918** [comment]
i did fuck it up one time but fixed it

**Comment by gyit918** [comment]
and the only reason i want to root is to hack apps dont ask why

**Comment by gyit918** [comment]
so what do i do now

**Comment by gyit918** [comment]
plz help me

**Comment by gyit918** [comment]
my apps still work

**Comment by gyit918** [comment]
> So can you puts all in a comment and don’t split them to separate comments

sure

**Comment by gyit918** [comment]
well anyone i have no root or do i what should it say for a success

**Comment by CerisWhite** [comment]
@gyit918 @vinaykharayat For the record, the correct command is `setenforce 0`, and the change doesn't apply to the selinux file. setenforce applies the change in RAM, and you can check the current status by doing `getenforce`

> Rooting does not have anything to do with SELinux enforcing or permissive.
As of <insert version of ChromeOS where android 9 began being shipped>, it's not possible to use a rooted image without SELinux being set to enforcing, so, yes, rooting has *everything* to do with SELinux being set to permissive.

If you have a question, please do not spam the thread with multiple small responses. Please make one large message with more info on what's going on so we can better assist you in the future.

**Comment by CerisWhite** [comment]
SELinux has to be set to permissive in order for the android system to work if cranberry.sh was successful. Otherwise, it will fail to load properly and apps will stay stuck in "loading" because the android image did not initialize properly.

TL;DR: We have to remove the security for root to work on ChromeOS, period.
